deps/build.jl

```diff
@@ -9,46 +9,66 @@ end
 
 function write_depsfile(path)
     open(depsfile,"w") do f
-        print(f,"const libcplex = ")
+        print(f, "const libcplex = ")
         show(f, path) # print with backslashes excaped on windows
         println(f)
     end
 end
 
-@static if (VERSION >= v"0.7.0-DEV.3382" && Sys.isapple()) || (VERSION < v"0.7.0-DEV.3382" && is_apple())
-    Libdl.dlopen("libstdc++",Libdl.RTLD_GLOBAL)
+@static if Sys.isapple()
+    Libdl.dlopen("libstdc++", Libdl.RTLD_GLOBAL)
 end
 
 base_env = "CPLEX_STUDIO_BINARIES"
 
+cplex_path = try
+        @static if Sys.isapple() || Sys.isunix()
+            read(`which cplex`, String)
+        elseif Sys.iswindows()
+            read(`where cplex`, String)
+        end
+    catch
+        nothing
+    end
+if cplex_path !== nothing
+    cplex_path = splitdir(strip(cplex_path))[1]
```

Comment:
Could you add a comment on usually what's the value of `cplex_path` after and before this line ?